_data-prepper/pipelines/configuration/processors/obfuscate.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: obfuscate
+parent: Processors
+grand_parent: Pipelines
+nav_order: 71
+---
+
+# obfuscate
+
+The `obfuscate` process enables obfuscation to fields inside your documents in order to protect sensitive data. 
+
+## Usage
+
+In this example, we have a document which contains a `log` field and a `phone` field that contains data we want to obfuscate, as shown in the following object:
+
+```json
+{
+  "id": 1,
+  "phone": "(555) 555 5555",
+  "log": "My name is Bob and my email address is abc@example.com"
+}
+```
+
+
+To obfuscate the `log` and `phone` fields multiple fields, add the `obfuscate` processor and call each field in the `source` option, as showing the following example. To account for both the `log` and `phone`, the following examples uses multiple `obfuscate` processors, since each processor can only obfuscate one source.
```

Comment:
```suggestion
To obfuscate the `log` and `phone` fields, add the `obfuscate` processor and call each field in the `source` option. To account for both the `log` and `phone` fields, the following example uses multiple `obfuscate` processors, since each processor can only obfuscate one source.
```